src/backend/marsha/core/tests/test_mail.py

```diff
@@ -0,0 +1,47 @@
+"""Test Emails."""
+from django.core import mail
+from django.template.loader import render_to_string
+from django.test import TestCase
+from django.utils.translation import gettext_lazy as _
+
+
+class EmailTestCase(TestCase):
```

Comment:
This test case is useless IMO. There is no need to test the django implementation, we can trust them. You will have to add tests once mail will be really sent in the application

Reply:
all of them you think, I get rid of the whole class?

Reply:
I think yes. But now you know how to test mail for your future tests :wink:

Reply:
lol